### ISSUE TITLE
Removed member re-appears in Cluster View

### ISSUE THREAD

**Issue by krgn** [dis-co/disco#15]
### Description

When removing a member from the active configuration, it disappears, only to re-appear moments later. The respective, removed service has in fact been removed and is idle.

### HowTo Reproduce

Remove a member from an active cluster configuration.

### Expected and actual results

I would exepect the removed member to stay "removed", i.e. not appear in the list of active members anymore.

### Related information

* Disco Version:
* RunTime Version:
* Operating system:

**Comment by krgn** [comment]
Yep, exactly. Its a race condition in the events that come from the raft sub system

**Comment by ingolfheinsch** [comment]
but if i reload the browser member is gone
